### PR TITLE
[4.2] users modal formatting

### PR DESCRIPTION
**Pull request by brianteeman**
Spotted this minor codestyle issue

code review only

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on d8d40f4fef96392a17ae3a687e0a298e66727dea<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39891">issues.joomla.org/tracker/joomla-cms/39891</a>.</sub>

File: administrator/components/com_users/tmpl/users/modal.php [hunk 1]
```diff
@@ -48,8 +48,8 @@
         <table class="table table-sm">
             <caption class="visually-hidden">
                 <?php echo Text::_('COM_USERS_USERS_TABLE_CAPTION'); ?>,
-                            <span id="orderedBy"><?php echo Text::_('JGLOBAL_SORTED_BY'); ?> </span>,
-                            <span id="filteredBy"><?php echo Text::_('JGLOBAL_FILTERED_BY'); ?></span>
+                <span id="orderedBy"><?php echo Text::_('JGLOBAL_SORTED_BY'); ?> </span>,
+                <span id="filteredBy"><?php echo Text::_('JGLOBAL_FILTERED_BY'); ?></span>
```
Comment (sandewt) [inline administrator/components/com_users/tmpl/users/modal.php:52]: Strange commas (,) at the and of line 50 and line 51

Comment (brianteeman) [reply administrator/components/com_users/tmpl/users/modal.php:52]: nothing strange at all

Comment (sandewt) [reply administrator/components/com_users/tmpl/users/modal.php:52]: issue 39891

Sorry, but Haven't seen this before in Joomla

Comment (richard67) [reply administrator/components/com_users/tmpl/users/modal.php:52]: @sandewt A caption is a text which may contain commas. A span is an inline element which can be in the middle of some text, so it is here. As the caption is visually hidden I assume its purpose is for screen readers, so the comma has some purpose (either a short break in reading or really read as a comma, @brianteeman will know better than I do).

Comment (richard67) [reply administrator/components/com_users/tmpl/users/modal.php:52]: P.S.: And the span with an id is used by some javascript to pick up the right part of the text and add the right (dynamic)  values after the colon, so a screen reader can tell how the table is ordered and so on. That's what I assume why it is like it is.

Comment (sandewt) [reply administrator/components/com_users/tmpl/users/modal.php:52]: Thank you for thinking along. I think you should have an open critical attitude when testing.

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on d8d40f4fef96392a17ae3a687e0a298e66727dea<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39891">issues.joomla.org/tracker/joomla-cms/39891</a>.</sub>

File: administrator/components/com_users/tmpl/users/modal.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_users/tmpl/users/modal.php:52]: Or you could have looked at the thirty plus examples of the same code

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39891">issues.joomla.org/tracker/joomla-cms/39891</a>.</sub>

File: administrator/components/com_users/tmpl/users/modal.php (hunk 1 above)
Comment (sandewt) [reply administrator/components/com_users/tmpl/users/modal.php:52]: Oops

**Comment by roland-d** [comment]
Thank you

**Comment by brianteeman** [comment]
thanks

**Pull request merged**